### PR TITLE
remove direct, unversioned dependency on json gem

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
should fix the tests

File: Gemfile [hunk 1]
```diff
@@ -34,7 +34,6 @@ gem 'mime-types', '~> 1.0', {"platforms"=>["ruby_18"], "groups"=>["development"]
 gem 'json', '~> 1.0', {"platforms"=>["ruby_18", "ruby_19"], "groups"=>["test"]}
```
Comment (mmoll) [inline Gemfile]: this is coming from modulesync

**Comment by domcleal** [comment]
Could you also remove the json item from .sync.yml, if this passes?

**Comment by mmoll** [comment]
There was a green build for Ruby 1.9 and Ruby 2.0 each, now removed it from .sync.yml

**Comment by ekohl** [comment]
:+1: on green tests

**Comment by domcleal** [comment]
Merged, thanks @mmoll.

**Pull request merged**